tex/generic/pgf/libraries/pgflibrarysvg.path.code.tex

```diff
@@ -12,6 +12,9 @@
 
 \usepgfmodule{parser}%
 
+% pgfmath is needed to parse scientific notation
+\input pgfmath.code.tex
```

Comment:
It does no harm inputting it twice `pgfmath.code.tex` has safety measures against that, and I wasn't sure whether it will be loaded in every case this library might be loaded.